internal/scm/github/github.go

```diff
@@ -357,6 +358,31 @@ func (g *Github) GetPullRequests(ctx context.Context, branchName string) ([]scm.
 		return nil, err
 	}
 
+	wg, ctx := errgroup.WithContext(ctx)
+	batches := chunkSlice(repos, 50)
+	var pullRequests []scm.PullRequest
+
```

Comment:
We should add a comment describing why all PRs are not fetched with one request.